### PR TITLE
Update "team" and "project" page

### PR DESCRIPTION
**Pull request by hikes101**
Remove Kevin from Team page and Bildungscampus from Projekte page

### CODE REVIEW

**Comment by hikes101** [comment]
[Projekte: Remove Bildungscampus Treptow-Nord](https://trello.com/c/RFVtNaUS/147-projekte-remove-bildungscampus-treptow-nord)

**Comment by hikes101** [comment]
[Menschen: Remove Kevin Machura](https://trello.com/c/pIVxHZsL/150-menschen-remove-kevin-machura)

File: data/team.json [hunk 1]
```diff
@@ -24,12 +24,6 @@
       "img": "img/team/michaela.jpg",
       "description": "Seit Michaela begonnen hat Nachhilfe zu geben konnte sie sich nicht mehr davon lösen. Sie hat sehr viel Freude daran Schüler zu unterrichten und zu unterstützen. Im sportlichen Bereich ist sie ebenso engagiert und ist ehrenamtlich als Trainerin in einem Kung Fu Verein tätig. Die Lernwerkstatt sieht sie als Chance allen Kindern und Jugendlichen unabhängig von finanziellen Mitteln die Möglichkeit zu geben, eine Lernbegleitung in Anspruch zu nehmen. Iht Ziel ist es, auch außerhalb der Schule eine Begeisterung fürs Lernen ins Leben zu rufen. Neben dem Unterrichten gehört auch Reisen zu ihren großen Leidenschaften."
     },
-    {
-      "name": "Kevin Machura",
-      "id": "kevin",
-      "img": "img/team/kevin.jpg",
```
Comment (pheeria) [inline data/team.json]: And also this image as well :)

Comment (hikes101) [reply data/team.json]: Done

**Pull request merged**